src/eth_provider/provider.rs

```diff
@@ -625,7 +620,7 @@ where
         &self,
         block_id: Option<BlockId>,
     ) -> EthProviderResult<Option<Vec<reth_rpc_types::Transaction>>> {
-        let block_id = match block_id.unwrap_or(BlockId::Number(BlockNumberOrTag::Latest)) {
+        let block_id = match block_id.unwrap_or_default() {
```

Comment:
I don't know if I am a fan of that, I know  ok, but performance wise it won't change anything and this makes it more readable, you know straight what the default block id is

Reply:
Yes understood, this is right that for readability, old solution was better. This was to lighten the code, but let's stay like that, just reversed :)